manifests/l/LeagueOfSelect/Windows/1.0.0/LeagueOfSelect.Windows.locale.en-US.yaml

```
@@ -0,0 +1,17 @@
+# Created using wingetcreate 1.10.3.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.10.0.schema.json
+
+PackageIdentifier: LeagueOfSelect.Windows

```

```suggestion
PackageIdentifier: LeagueOfSelect.LeagueOfSelect
```
Please update file name and path to match this change.